.github/workflows/tests.yaml

```diff
@@ -28,11 +31,14 @@ jobs:
         fail_ci_if_error: true
         files: ./coverage.xml
         flags: unittests
-  Integration-Tests:
+
+  integration-tests:
+    name: Run Integration Tests
     runs-on: ubuntu-latest
+    needs: unit-tests
```

Comment:
Why not run them in parallel as before?

Reply:
I think running in parallel is better. Doesn't hurt and in case both are failing you see it immediately and not after the next push